docs/tools/vdb_table/data/rockset.json

```diff
@@ -134,13 +134,13 @@
   },
   "sharding": {
     "support": "full",
-    "source_url": "",
+    "source_url": "https://docs.rockset.com/documentation/docs/what-is-rockset",
```

Comment:
I see that this link mentions that Rockset "offloads the work of shard / index management". Is there any more detailed doc to show that Rockset is able to scale its vector indexes horizontally as data size grows?

Reply:
We discuss how Rockset indexes are distributed across data for vector search in this tech talk: https://www.youtube.com/watch?v=trXiMHjP6a8. You can read Rockset's architecture whitepaper, showing how we handle shard/index management across all use cases: https://rockset.com/whitepapers/rockset-concepts-designs-and-architecture

I have updated the URL and comment In the last commit
--


We discuss how Rockset indexes are distributed across data for vector search in this [tech talk](https://www.youtube.com/watch?v=trXiMHjP6a8). You can read Rockset's [architecture whitepaper](https://rockset.com/whitepapers/rockset-concepts-designs-and-architecture/), showing how we handle shard/index management across all use cases.

Reply:
Thanks for providing both the references!